lib/utils/token-util.ts

```diff
@@ -0,0 +1,16 @@
+import { createHash, randomBytes } from 'crypto';
+import { promisify } from 'util';
+const randomBytesPromise = promisify(randomBytes);
+
+/**
+ * Generate random token.
+ */
+
+export const GenerateRandomToken = async () => {
+  const bytesSize = 256;
+  const buffer = await randomBytesPromise(bytesSize);
+
+  return createHash('sha1')
```

Comment:
I forget if tokens HAVE TO or MUST be `sha1`; if not I wonder if that should be vaguely configurable or at least documented that's what they are in this implementation.

Reply:
this code just generated a random code with a fixed length.

Reply:
Also this is just the fallback if the used grant type adapter has no own generateAccessToken method. You could always implement your own grant-type adapter and use your own token generator 

See lib/grant-types/abstract-grant-type.ts line 36

So nothing wrong here